sections/futures/ProfitCalculator/ProfitDetails.tsx

```diff
@@ -86,7 +86,7 @@ const ProfitDetails: React.FC<ProfitDetailsProps> = ({
 							? wei(marketAssetPositionSize).toNumber().toFixed(2)
 							: ''}
 					</RowText>
-					<RowText className="gray-font-color">{`${marketAsset}-PERP`}</RowText>
+					<RowText className="gray-font-color">{marketAsset}</RowText>
```

Comment:
Did you mean to wrap this in `getMarketName`, or does the `marketAsset` value have the "-PERP" suffix already?

Reply:
Fixed this with a new commit